core/native/src/cta/cta_util_statistics.c

```diff
@@ -82,17 +85,17 @@ float ran2(long *idum)
         }
 }
 
-
+#define METHOD "CTA_rand_seed"
 void CTA_rand_seed(int seed){
-   printf("cta_util_statistics: Set initial seed: %ld\n",seed);
+   char message[64];
+   sprintf(message,"Set initial seed: %ld\n",seed);

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [%ld in format string (no. 1) requires 'long' but the argument type is 'signed int'.](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5593693)